eng/pipelines/tools-repo-versioning.yml

```diff
@@ -0,0 +1,42 @@
+# Tag the tools repos (azure-sdk-tools or azure-sdk-build-tools) to create some form of versioning.
+resources:
+  repositories:
+    - repository: azure-sdk-tools
+      type: github
+      name: Azure/azure-sdk-tools
+      endpoint: azure
+    - repository: azure-sdk-build-tools
+      type: git
+      name: internal/azure-sdk-build-tools
+
+trigger: none
+
+pr: none
+
+pool:
+  vmImage: windows-2019
+
+jobs:
+  - job: TagToolsRepo
+    displayName: Version Repo with Git Tag
+
+    steps:
+    - checkout: azure-sdk-tools
+    - checkout: azure-sdk-build-tools
+
+    - pwsh: |
+        if ("$(RepoType)" -eq "github")
```

Comment:
Remove this and simply hard code based on what repo we are tagging and setup a new pipeline for each. So for azure-sdk-tools this would just be:
```
$PushURL = "https://$(azure-sdk-github-pat)@github.com/Azure/azure-sdk-tools.git"
```

When you setup the pipeline hook-up the variable group that pulls in the pat.